script

```diff
@@ -0,0 +1,16 @@
+docker build -t svalinn/test_dagmc-ci-ubuntu-16.04-clang-g4-hdf5_1.10.4-moab_master --build-arg UBUNTU_VERSION=16.04 --build-arg COMPILER=clang --build-arg HDF5=1.10.4 --build-arg MOAB=master -f CI/Dockerfile_3_moab .
```

Comment:
Is this meant to be part of the PR?  Doesn't it duplicate what's in `update_docker.sh`?

Reply:
I don't think that was not supposed to be in there.

Reply:
was your double negative in the above sentence intentional or French?

Reply:
Neither I edited my sentence and forgot the second negative...